tests/AmpTest.php

```diff
@@ -1,15 +1,11 @@
 <?php
 
-namespace AmpProject\Common;
+namespace AmpProject;
```

Comment:
Shouldn't this file be in `tests/AmpProject`?

Reply:
The tests themselves are not autoloaded. The dev-autoloader that we're using points to the `src` subfolder, and is used for complementary files that are needed as dependencies of the tests, but the test files are directly loaded by PHPUnit.

Here, the namespace is only used to define in which "context" we want to run the tests, so we don't need huge lists of imports for the tested itself.

Reply:
Got it, thanks :+1:.